server/aws-lsp-identity/src/language-server/identityService.ts

```diff
@@ -0,0 +1,6 @@
+import { SsoTokenChangedParams } from '@aws/language-server-runtimes/protocol'
```

Comment:
nit: `SsoTokenChangedParams` can be imported from 'server-interface'

Reply:
Updated, will appear in the next PR.